editor/plugins/animation_library_editor.cpp

```diff
@@ -35,9 +35,10 @@
 #include "editor/editor_string_names.h"
 #include "editor/editor_undo_redo_manager.h"
 #include "editor/gui/editor_file_dialog.h"
+#include "scene/animation/animation_mixer.h"
 
 void AnimationLibraryEditor::set_animation_mixer(Object *p_mixer) {
-	mixer = p_mixer;
+	mixer = Object::cast_to<AnimationMixer>(p_mixer);
 }
```

Comment:
I'm concerned we don't handle null anywhere, which it can be (even without a cast). But that's already present in this code, so this PR doesn't make things worse.